lib/utils.pm

```diff
@@ -298,6 +299,8 @@ sub clear_console {
     type_string "clear\n";
 }
 
+sub desktop_runner_hotkey { check_var('DESKTOP', 'minimalx') ? 'super-spc' : 'alt-f2' }
```

Comment:
(oops. Sorry, first I edited your comment instead of adding my own).

Moving to "x11test" would only work if I also move the method `ensure_unlocked_desktop` there. A separate lib-file we could do when we know other methods that could fit there as well. I will take a look.

Reply:
yeah I think better to do this in separate PR ...

Reply:
@rwx788 I am not it to move some methods out ouf "lib/utils" but it will be a bigger change so if you don't mind I would like to have this fix merged earlier, then we can continue in another PR, ok?